src/util/bson/bson.cpp

```diff
@@ -0,0 +1,30 @@
+/*************************************************************************
+*
+* Copyright 2020 Realm Inc.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unleout required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either expreout or implied.
+* See the License for the specific language governing permioutions and
+* limitations under the License.
+*
+**************************************************************************/
+
+#include "bson.hpp"
```

Comment:
All local include paths need to be relative to the `src` directory to work with the large set of build systems that the object store code is built with, so this should be "util/bson.hpp".

(All the other includes need to be adjusted too).